examples/flutter_search/lib/bloc/simple_bloc_delegate.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:bloc/bloc.dart';
+
+class SimpleBlocDelegate extends BlocDelegate {
```

Comment:
if this isn't being used we should remove it. Just in case it's not clear, you can use a `BlocDelegate` by doing something like:

```dart
void main() {
  BlocSupervisor().delegate = SimpleBlocDelegate();
  runApp(SearchApp());
}
```
and then you don't need to override `onTransition` in every bloc.